tests/unit/models/ObjectMDArchive.spec.js

```diff
@@ -0,0 +1,109 @@
+const assert = require('assert');
+const ObjectMDArchive = require('../../../lib/models/ObjectMDArchive');
+
+const testArchive = {
+    archiveInfo: {
+        any: 'value',
+    },
+    restoreRequestedAt: new Date(0),
+    restoreRequestedDays: 5,
+    restoreCompletedAt: new Date(1000),
+    restoreWillExpireAt: new Date(10000),
+};
+
+const archive = new ObjectMDArchive(
+    testArchive.archiveInfo,
+    testArchive.restoreRequestedAt,
+    testArchive.restoreRequestedDays,
+    testArchive.restoreCompletedAt,
+    testArchive.restoreWillExpireAt,
+);
+
+describe('ObjectMDArchive value', () => {
+    it('should return the correct value', () => {
+        const amzRestoreObj = archive.getValue();
+        assert.deepStrictEqual(amzRestoreObj, archive);
+    });
+});
+
+describe('ObjectMDArchive setters/getters', () => {
+    let archived = null;
+    beforeEach(() => {
+        archived = new ObjectMDArchive(
+            testArchive.archiveInfo,
+            undefined,
+            undefined,
+            undefined,
+            undefined,
```

Comment:
Same here, the function has de facto variable arity so no need to fill in the blanks with `undefined`s, it defeats the purpose.

Reply:
Done in https://github.com/scality/Arsenal/pull/1856/commits/11106e9db5c2a08f652d9ebf5564020125678863#diff-4a93125ecd0054d2891dd3ea883b3012792a1a1a05594172ed4383a0ddf80eaaL34